--- libgav1-0.18.0/CMakeLists.txt.orig	2023-03-05 17:47:49.798480620 +0100
+++ libgav1-0.18.0/CMakeLists.txt	2023-03-05 17:48:09.008376551 +0100
@@ -16,8 +16,8 @@
 cmake_minimum_required(VERSION 3.7.1 FATAL_ERROR)
 
 # libgav1 requires C++11.
-set(CMAKE_CXX_STANDARD 11)
-set(ABSL_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
+set(ABSL_CXX_STANDARD 17)
 # libgav1 requires C99.
 set(CMAKE_C_STANDARD 99)
 
