--- libgav1-0.18.0/CMakeLists.txt.orig	2023-03-05 17:34:54.142000000 +0100
+++ libgav1-0.18.0/CMakeLists.txt	2023-03-05 17:39:16.107930187 +0100
@@ -116,9 +116,6 @@ if(LIBGAV1_VERBOSE)
   libgav1_dump_options()
 endif()
 
-set(libgav1_abseil_build "${libgav1_build}/abseil")
-set(libgav1_gtest_build "${libgav1_build}/gtest")
-
 # Compiler/linker flags must be lists, but come in from the environment as
 # strings. Break them up:
 if(NOT "${LIBGAV1_CXX_FLAGS}" STREQUAL "")
@@ -131,24 +128,6 @@ endif()
 # Set test-only flags based on LIBGAV1_CXX_FLAGS.
 libgav1_set_test_flags()
 
-set(libgav1_abseil "${libgav1_root}/third_party/abseil-cpp")
-if(EXISTS "${libgav1_abseil}")
-  set(ABSL_PROPAGATE_CXX_STD ON)
-  add_subdirectory("${libgav1_abseil}" "${libgav1_abseil_build}"
-                   EXCLUDE_FROM_ALL)
-else()
-  if(use_absl_threading OR LIBGAV1_ENABLE_EXAMPLES OR LIBGAV1_ENABLE_TESTS)
-    message(
-      FATAL_ERROR
-        "Abseil not found. This dependency is required by the"
-        " examples & tests and libgav1 when LIBGAV1_THREADPOOL_USE_STD_MUTEX is"
-        " not defined. To continue, download the Abseil repository to"
-        " third_party/abseil-cpp:\n  git \\\n    -C ${libgav1_root} \\\n"
-        "    clone \\\n"
-        "    https://github.com/abseil/abseil-cpp.git third_party/abseil-cpp")
-  endif()
-endif()
-
 libgav1_reset_target_lists()
 libgav1_add_dsp_targets()
 libgav1_add_decoder_targets()
--- libgav1-0.16.3/examples/libgav1_examples.cmake.orig	2021-06-30 16:32:33.000000000 +0200
+++ libgav1-0.16.3/examples/libgav1_examples.cmake	2021-06-30 16:35:57.392250015 +0200
@@ -56,8 +56,8 @@ macro(libgav1_add_examples_targets)
                          libgav1_file_reader
                          libgav1_file_writer
                          LIB_DEPS
-                         absl::strings
-                         absl::str_format_internal
-                         absl::time
+                         absl_strings
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_dependency})
 endmacro()
--- libgav1-0.18.0/src/libgav1_decoder.cmake.orig	2023-03-05 17:34:54.305000000 +0100
+++ libgav1-0.18.0/src/libgav1_decoder.cmake	2023-03-05 17:42:32.716865065 +0100
@@ -108,7 +108,7 @@ macro(libgav1_add_decoder_targets)
   endif()
 
   if(use_absl_threading)
-    list(APPEND libgav1_absl_deps absl::base absl::synchronization)
+    list(APPEND libgav1_absl_deps absl_base absl_synchronization)
   endif()
 
   libgav1_add_library(NAME libgav1_decoder TYPE OBJECT SOURCES
@@ -151,6 +151,7 @@ macro(libgav1_add_decoder_targets)
                         ${libgav1_include_paths}
                         LIB_DEPS
                         libgav1_static
+                        pthread
                         PUBLIC_INCLUDES
                         ${libgav1_source})
   endif()
--- libgav1-0.18.0/tests/libgav1_tests.cmake.orig	2023-03-05 17:34:54.422000000 +0100
+++ libgav1-0.18.0/tests/libgav1_tests.cmake	2023-03-05 17:47:32.268575588 +0100
@@ -17,24 +17,6 @@ if(LIBGAV1_LIBGAV1_TESTS_CMAKE_)
 endif() # LIBGAV1_LIBGAV1_TESTS_CMAKE_
 set(LIBGAV1_LIBGAV1_TESTS_CMAKE_ 1)
 
-set(libgav1_googletest "${libgav1_root}/third_party/googletest")
-if(NOT LIBGAV1_ENABLE_TESTS OR NOT EXISTS "${libgav1_googletest}")
-  macro(libgav1_add_tests_targets)
-
-  endmacro()
-
-  if(LIBGAV1_ENABLE_TESTS AND NOT EXISTS "${libgav1_googletest}")
-    message(
-      "GoogleTest not found, setting LIBGAV1_ENABLE_TESTS to false.\n"
-      "To enable tests download the GoogleTest repository to"
-      " third_party/googletest:\n\n  git \\\n    -C ${libgav1_root} \\\n"
-      "    clone \\\n"
-      "    https://github.com/google/googletest.git third_party/googletest\n")
-    set(LIBGAV1_ENABLE_TESTS FALSE CACHE BOOL "Enables tests." FORCE)
-  endif()
-  return()
-endif()
-
 # Check GoogleTest compiler requirements.
 if((CMAKE_CXX_COMPILER_ID
     MATCHES
@@ -195,34 +177,8 @@ macro(libgav1_add_tests_targets)
         "This version of libgav1_add_tests_targets() should only be used with"
         " LIBGAV1_ENABLE_TESTS set to true.")
   endif()
-  libgav1_add_library(TEST
-                      NAME
-                      libgav1_gtest
-                      TYPE
-                      STATIC
-                      SOURCES
-                      "${libgav1_googletest}/googletest/src/gtest-all.cc"
-                      DEFINES
-                      ${libgav1_defines}
-                      INCLUDES
-                      ${libgav1_gtest_include_paths}
-                      ${libgav1_include_paths})
-
-  libgav1_add_library(TEST
-                      NAME
-                      libgav1_gtest_main
-                      TYPE
-                      STATIC
-                      SOURCES
-                      "${libgav1_googletest}/googletest/src/gtest_main.cc"
-                      DEFINES
-                      ${libgav1_defines}
-                      INCLUDES
-                      ${libgav1_gtest_include_paths}
-                      ${libgav1_include_paths})
-
   if(use_absl_threading)
-    list(APPEND libgav1_common_test_absl_deps absl::synchronization)
+    list(APPEND libgav1_common_test_absl_deps absl_synchronization absl_time)
   endif()
 
   libgav1_add_executable(TEST
@@ -238,8 +194,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -254,8 +210,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -269,10 +225,10 @@ macro(libgav1_add_tests_targets)
                          OBJLIB_DEPS
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   if(libgav1_have_avx2)
     libgav1_add_executable(TEST
@@ -286,8 +242,8 @@ macro(libgav1_add_tests_targets)
                            ${libgav1_test_include_paths}
                            LIB_DEPS
                            ${libgav1_common_test_absl_deps}
-                           libgav1_gtest
-                           libgav1_gtest_main)
+                           gtest
+                           gtest_main)
   endif()
 
   if(libgav1_have_neon)
@@ -304,8 +260,8 @@ macro(libgav1_add_tests_targets)
                            libgav1_tests_block_utils
                            LIB_DEPS
                            ${libgav1_common_test_absl_deps}
-                           libgav1_gtest
-                           libgav1_gtest_main)
+                           gtest
+                           gtest_main)
   endif()
 
   if(libgav1_have_sse4)
@@ -320,8 +276,8 @@ macro(libgav1_add_tests_targets)
                            ${libgav1_test_include_paths}
                            LIB_DEPS
                            ${libgav1_common_test_absl_deps}
-                           libgav1_gtest
-                           libgav1_gtest_main)
+                           gtest
+                           gtest_main)
   endif()
 
   libgav1_add_executable(TEST
@@ -337,8 +293,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -353,8 +309,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -368,10 +324,10 @@ macro(libgav1_add_tests_targets)
                          OBJLIB_DEPS
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -388,11 +344,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          libgav1_tests_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -407,10 +363,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_file_reader
                          libgav1_utils
                          LIB_DEPS
-                         absl::memory
+
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -428,11 +384,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -444,9 +400,9 @@ macro(libgav1_add_tests_targets)
                          INCLUDES
                          ${libgav1_test_include_paths}
                          LIB_DEPS
-                         absl::base
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         absl_base
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -461,8 +417,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -477,8 +433,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -493,8 +449,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -509,8 +465,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -527,8 +483,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -542,9 +498,9 @@ macro(libgav1_add_tests_targets)
                          OBJLIB_DEPS
                          libgav1_utils
                          LIB_DEPS
-                         absl::synchronization
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         absl_synchronization absl_time
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -559,8 +515,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -576,11 +532,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -595,8 +551,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -610,8 +566,8 @@ macro(libgav1_add_tests_targets)
                          LIB_DEPS
                          ${libgav1_dependency}
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_library(TEST
                       NAME
@@ -653,11 +609,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -674,8 +630,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -692,11 +648,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -714,11 +670,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -756,8 +713,8 @@ macro(libgav1_add_tests_targets)
                          LIB_DEPS
                          ${libgav1_dependency}
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -771,8 +728,8 @@ macro(libgav1_add_tests_targets)
                          LIB_DEPS
                          ${libgav1_dependency}
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -789,11 +746,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -810,11 +767,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -832,12 +789,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::memory
-                         absl::strings
-                         absl::time
+
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -855,10 +812,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -876,10 +833,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -897,10 +854,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -918,10 +875,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -938,11 +895,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_dsp
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -960,11 +917,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_dsp
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -981,8 +938,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1000,10 +957,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1021,10 +978,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1041,11 +998,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1062,11 +1019,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1083,11 +1041,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1105,11 +1063,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1126,8 +1084,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1145,10 +1103,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1165,11 +1123,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1186,8 +1144,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1202,8 +1160,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1222,10 +1180,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
-                         absl::strings
+                         absl_strings
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1243,8 +1201,8 @@ macro(libgav1_add_tests_targets)
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1260,8 +1218,8 @@ macro(libgav1_add_tests_targets)
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1278,11 +1236,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1299,10 +1257,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
-                         absl::str_format_internal
+                         absl_str_format_internal absl_strings
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1320,11 +1278,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1341,8 +1300,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1359,9 +1318,9 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 endmacro()
