--- libgav1-0.19.0/CMakeLists.txt.orig	2024-03-26 11:13:01.333000000 +0100
+++ libgav1-0.19.0/CMakeLists.txt	2024-03-26 18:00:12.304514039 +0100
@@ -116,9 +116,6 @@ if(LIBGAV1_VERBOSE)
   libgav1_dump_options()
 endif()
 
-set(libgav1_abseil_build "${libgav1_build}/abseil")
-set(libgav1_gtest_build "${libgav1_build}/gtest")
-
 # Compiler/linker flags must be lists, but come in from the environment as
 # strings. Break them up:
 if(NOT "${LIBGAV1_CXX_FLAGS}" STREQUAL "")
@@ -131,24 +128,6 @@ endif()
 # Set test-only flags based on LIBGAV1_CXX_FLAGS.
 libgav1_set_test_flags()
 
-set(libgav1_abseil "${libgav1_root}/third_party/abseil-cpp")
-if(EXISTS "${libgav1_abseil}")
-  set(ABSL_PROPAGATE_CXX_STD ON)
-  add_subdirectory("${libgav1_abseil}" "${libgav1_abseil_build}"
-                   EXCLUDE_FROM_ALL)
-else()
-  if(use_absl_threading OR LIBGAV1_ENABLE_EXAMPLES OR LIBGAV1_ENABLE_TESTS)
-    message(
-      FATAL_ERROR
-        "Abseil not found. This dependency is required by the"
-        " examples & tests and libgav1 when LIBGAV1_THREADPOOL_USE_STD_MUTEX is"
-        " not defined. To continue, download the Abseil repository to"
-        " third_party/abseil-cpp:\n  git \\\n    -C ${libgav1_root} \\\n"
-        "    clone -b 20220623.0 --depth 1 \\\n"
-        "    https://github.com/abseil/abseil-cpp.git third_party/abseil-cpp")
-  endif()
-endif()
-
 libgav1_reset_target_lists()
 libgav1_add_dsp_targets()
 libgav1_add_decoder_targets()
--- libgav1-0.16.3/examples/libgav1_examples.cmake.orig	2021-06-30 16:32:33.000000000 +0200
+++ libgav1-0.16.3/examples/libgav1_examples.cmake	2021-06-30 16:35:57.392250015 +0200
@@ -56,8 +56,8 @@ macro(libgav1_add_examples_targets)
                          libgav1_file_reader
                          libgav1_file_writer
                          LIB_DEPS
-                         absl::strings
-                         absl::str_format_internal
-                         absl::time
+                         absl_strings
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_dependency})
 endmacro()
--- libgav1-0.18.0/src/libgav1_decoder.cmake.orig	2023-03-05 17:34:54.305000000 +0100
+++ libgav1-0.18.0/src/libgav1_decoder.cmake	2023-03-05 17:42:32.716865065 +0100
@@ -108,7 +108,7 @@ macro(libgav1_add_decoder_targets)
   endif()
 
   if(use_absl_threading)
-    list(APPEND libgav1_absl_deps absl::base absl::synchronization)
+    list(APPEND libgav1_absl_deps absl_base absl_synchronization)
   endif()
 
   libgav1_add_library(NAME libgav1_decoder TYPE OBJECT SOURCES
@@ -151,6 +151,7 @@ macro(libgav1_add_decoder_targets)
                         ${libgav1_include_paths}
                         LIB_DEPS
                         libgav1_static
+                        pthread
                         PUBLIC_INCLUDES
                         ${libgav1_source})
   endif()
--- libgav1-0.20.0/tests/libgav1_tests.cmake.orig	2025-02-22 18:02:44.152000000 +0100
+++ libgav1-0.20.0/tests/libgav1_tests.cmake	2025-02-22 19:09:05.741879887 +0100
@@ -17,24 +17,6 @@ if(LIBGAV1_LIBGAV1_TESTS_CMAKE_)
 endif() # LIBGAV1_LIBGAV1_TESTS_CMAKE_
 set(LIBGAV1_LIBGAV1_TESTS_CMAKE_ 1)
 
-set(libgav1_googletest "${libgav1_root}/third_party/googletest")
-if(NOT LIBGAV1_ENABLE_TESTS OR NOT EXISTS "${libgav1_googletest}")
-  macro(libgav1_add_tests_targets)
-
-  endmacro()
-
-  if(LIBGAV1_ENABLE_TESTS AND NOT EXISTS "${libgav1_googletest}")
-    message(
-      "GoogleTest not found, setting LIBGAV1_ENABLE_TESTS to false.\n"
-      "To enable tests download the GoogleTest repository to"
-      " third_party/googletest:\n\n  git \\\n    -C ${libgav1_root} \\\n"
-      "    clone -b release-1.12.1 --depth 1 \\\n"
-      "    https://github.com/google/googletest.git third_party/googletest\n")
-    set(LIBGAV1_ENABLE_TESTS FALSE CACHE BOOL "Enables tests." FORCE)
-  endif()
-  return()
-endif()
-
 # Check GoogleTest compiler requirements.
 if((CMAKE_CXX_COMPILER_ID
     MATCHES
@@ -202,34 +184,8 @@ macro(libgav1_add_tests_targets)
         "This version of libgav1_add_tests_targets() should only be used with"
         " LIBGAV1_ENABLE_TESTS set to true.")
   endif()
-  libgav1_add_library(TEST
-                      NAME
-                      libgav1_gtest
-                      TYPE
-                      STATIC
-                      SOURCES
-                      "${libgav1_googletest}/googletest/src/gtest-all.cc"
-                      DEFINES
-                      ${libgav1_defines}
-                      INCLUDES
-                      ${libgav1_gtest_include_paths}
-                      ${libgav1_include_paths})
-
-  libgav1_add_library(TEST
-                      NAME
-                      libgav1_gtest_main
-                      TYPE
-                      STATIC
-                      SOURCES
-                      "${libgav1_googletest}/googletest/src/gtest_main.cc"
-                      DEFINES
-                      ${libgav1_defines}
-                      INCLUDES
-                      ${libgav1_gtest_include_paths}
-                      ${libgav1_include_paths})
-
   if(use_absl_threading)
-    list(APPEND libgav1_common_test_absl_deps absl::synchronization)
+    list(APPEND libgav1_common_test_absl_deps absl_synchronization absl_time)
   endif()
 
   libgav1_add_executable(TEST
@@ -245,8 +201,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -261,8 +217,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -276,10 +232,10 @@ macro(libgav1_add_tests_targets)
                          OBJLIB_DEPS
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   if(libgav1_have_avx2)
     list(APPEND libgav1_common_dsp_test_sources
@@ -303,8 +259,8 @@ macro(libgav1_add_tests_targets)
                            libgav1_utils
                            LIB_DEPS
                            ${libgav1_common_test_absl_deps}
-                           libgav1_gtest_main
-                           libgav1_gtest)
+                           gtest_main
+                           gtest)
   endif()
 
   if(libgav1_have_neon)
@@ -322,8 +278,8 @@ macro(libgav1_add_tests_targets)
                            libgav1_utils
                            LIB_DEPS
                            ${libgav1_common_test_absl_deps}
-                           libgav1_gtest
-                           libgav1_gtest_main)
+                           gtest
+                           gtest_main)
   endif()
 
   libgav1_add_executable(TEST
@@ -339,8 +295,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -355,8 +311,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -370,10 +326,10 @@ macro(libgav1_add_tests_targets)
                          OBJLIB_DEPS
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -390,11 +346,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          libgav1_tests_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -409,10 +365,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_file_reader
                          libgav1_utils
                          LIB_DEPS
-                         absl::memory
+
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -430,11 +386,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -446,9 +402,9 @@ macro(libgav1_add_tests_targets)
                          INCLUDES
                          ${libgav1_test_include_paths}
                          LIB_DEPS
-                         absl::base
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         absl_base
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -463,8 +419,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -479,8 +435,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -495,8 +451,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -511,8 +467,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -529,8 +485,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -544,9 +500,9 @@ macro(libgav1_add_tests_targets)
                          OBJLIB_DEPS
                          libgav1_utils
                          LIB_DEPS
-                         absl::synchronization
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         absl_synchronization absl_time
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -561,8 +517,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -578,11 +534,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -597,8 +553,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -612,8 +568,8 @@ macro(libgav1_add_tests_targets)
                          LIB_DEPS
                          ${libgav1_dependency}
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_library(TEST
                       NAME
@@ -655,11 +611,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -676,8 +632,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -694,11 +650,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -716,11 +672,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -758,8 +715,8 @@ macro(libgav1_add_tests_targets)
                          LIB_DEPS
                          ${libgav1_dependency}
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -773,8 +730,8 @@ macro(libgav1_add_tests_targets)
                          LIB_DEPS
                          ${libgav1_dependency}
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -791,11 +748,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -812,11 +769,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -834,12 +791,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::memory
-                         absl::strings
-                         absl::time
+
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -857,10 +814,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -878,10 +835,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -899,10 +856,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -920,10 +877,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -940,11 +897,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_dsp
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -962,11 +919,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_dsp
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -983,8 +940,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1002,10 +959,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1023,10 +980,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1043,11 +1000,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1064,11 +1021,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1085,11 +1043,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1107,11 +1065,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1128,8 +1086,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1147,10 +1105,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::time
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1167,11 +1125,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1188,8 +1146,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1204,8 +1162,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1224,11 +1182,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
-                         absl::strings
-                         absl::time
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1246,8 +1204,8 @@ macro(libgav1_add_tests_targets)
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1263,8 +1221,8 @@ macro(libgav1_add_tests_targets)
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1281,11 +1239,11 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1302,10 +1260,10 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          ${libgav1_test_objlib_deps}
                          LIB_DEPS
-                         absl::str_format_internal
+                         absl_str_format_internal absl_strings
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1323,11 +1281,12 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_strings
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1344,8 +1303,8 @@ macro(libgav1_add_tests_targets)
                          libgav1_utils
                          LIB_DEPS
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 
   libgav1_add_executable(TEST
                          NAME
@@ -1362,9 +1321,9 @@ macro(libgav1_add_tests_targets)
                          libgav1_tests_utils
                          libgav1_utils
                          LIB_DEPS
-                         absl::str_format_internal
-                         absl::time
+                         absl_str_format_internal
+                         absl_time
                          ${libgav1_common_test_absl_deps}
-                         libgav1_gtest
-                         libgav1_gtest_main)
+                         gtest
+                         gtest_main)
 endmacro()
